src/ore/src/stats.rs

```diff
@@ -20,9 +20,10 @@
 /// see `histogram_seconds_buckets` below.
 ///
 /// Note that any changes to this range may modify buckets for existing metrics.
-const HISTOGRAM_SECOND_BUCKETS: [f64; 23] = [
+const HISTOGRAM_SECOND_BUCKETS: [f64; 31] = [
     0.000_008, 0.000_016, 0.000_032, 0.000_064, 0.000_128, 0.000_256, 0.000_512, 0.001, 0.002,
-    0.004, 0.008, 0.016, 0.032, 0.064, 0.128, 0.256, 0.512, 1.0, 2.0, 4.0, 8.0, 16.0, 32.0,
+    0.004, 0.008, 0.016, 0.032, 0.064, 0.128, 0.256, 0.512, 1.0, 2.0, 4.0, 8.0, 16.0, 32.0, 64.0,
+    128.0, 256.0, 512.0, 1024.0, 2048.0, 4096.0, 8192.0,
```

Comment:
(I've checked that the only existing metric that this change affected is one of the metrics that I added yesterday: `mz_pgwire_message_processing_seconds`. I didn't know that there is this limitation when I added that metric, so what I put into the `histogram_seconds_buckets` was not taking effect. Now this is fixed.)